static/app/data/platforms.tsx

```diff
@@ -25,7 +25,7 @@ const platformIntegrations: PlatformIntegration[] = [
 ]
   .map(platform => {
     const integrations = platform.integrations.reduce((acc, value) => {
-      // filter out any javascript-[angular|angularjs|ember|gatsby|nextjs|react|remix|svelte|vue]-* platforms; as they're not meant to be used as a platform in the PlatformPicker component
+      // filter out any javascript-[angular|angularjs|ember|gatsby|nextjs|react|remix|svelte|sveltekit|vue]-* platforms; as they're not meant to be used as a platform in the PlatformPicker component
```

Comment:
this is great!..but... I don't think we have docs for sveltekit in our wizard folder https://github.com/getsentry/sentry-docs/tree/master/src/wizard/javascript

Reply:
We'll soon have them https://github.com/getsentry/sentry-docs/pull/6867 :)

I'll merge this PR only after the docs PR was merged to make sure this isn't causing problems.

Reply:
https://github.com/getsentry/sentry-docs/pull/6867 is merged, I think this should be good to go now